elastic/synthetics-recorder
# fix: use default entitlements

## Summary

There has been a case some of the functionalities don't work since #290 is merged
The only part that can affect this seems to be updated entitlements.
I will revert back to the previous setting which uses the default entitlements list provided by [electron-osx-sign](https://github.com/electron/osx-sign/pull/253)

## How to validate this change

Build the apps from this branch and test all of the mac artifacts, then verify`Test` feature works
